doc/dev/GETTING_STARTED.md

```diff
@@ -86,9 +86,14 @@ mkdir build
 
  * Open git bash
 
+Check the value of `builtin-baseline` in the `vcpkg.json` file at the root of f3d source directory.  
+Make sure it's `c9140a3b500812ad3206317885860d9553b93f13` (please let us know if it's not)
```

Comment:
I don't think that comment is important? As long as we ensure the hash in the guide is the same as the one in the JSON file unit will work, right ?

Reply:
We may forgot right? There's nothing in the CI that ensure it's the case

Reply:
Can we come up with a bash command to read a value in a json file and check if the value is found in the documentation file?

Reply:
To me its similar to forgetting that archlinux dependencies are optional for vtk, we just need to put good info in the guide. If we change it in one location, we need to change it in the other location. Its just doc.

Reply:
Fair enough. I'll remove the comment, but you have been warned ^^

Reply:
move this forward ?

Reply:
done